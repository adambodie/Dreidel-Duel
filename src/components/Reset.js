import React, { Component } from "react";
import styles from "../css/main.module.scss";

export default class Reset extends Component {
  constructor(props) {
    super(props);
    this.handleChange = this.handleChange.bind(this);
  }

  handleChange(e) {
	this.props.onResetChange(e.target.value);  
  }

  render() {
    return (
          <button className={styles.action} onClick={this.handleChange} disabled={this.props.disabled} >
            Reset
          </button>
    );
  }
}
