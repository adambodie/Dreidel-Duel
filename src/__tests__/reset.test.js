import React from "react"
import renderer from "react-test-renderer"
import Reset from "../components/reset"

describe("Reset", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<Reset />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
