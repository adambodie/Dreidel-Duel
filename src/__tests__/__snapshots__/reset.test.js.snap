// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset renders correctly 1`] = `
<button
  className="action"
  onClick={[Function]}
>
  Reset
</button>
`;
